signup.php

```diff
@@ -3,38 +3,31 @@
 	<head>
 		<title>SignUp</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
-		<link rel ="stylesheet" href = "style8.css"> 
+		<!--Connected to css -->
+		<link rel ="stylesheet" href = "style.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
 				<a href = "01_login_real.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
-			<nav>
```

Comment:
04/08/2021: I removed this nav because I will use a difference way to called up the nav.